portal-api/api-tests/v1/gef/application.api-test.js

```diff
@@ -209,7 +197,7 @@ describe(baseUrl, () => {
       expect(body).toEqual([{
         errCode: 'MANDATORY_FIELD',
         errRef: 'bankInternalRefName',
-        errMsg: 'Application Reference Name is Mandatory',
+        errMsg: 'bankInternalRefName is Mandatory',
```

Comment:
is this `bankInternalRefName` displayed to users? if so, do they know what it means?
I'm thinking that the previous text was more readable (i.e. 'Application Reference Name')

Reply:
this msg is just from the api ie a technical message. The UI msg is mapped from the error code returned by the api.